app/southwest.py

```diff
@@ -1,14 +1,23 @@
-from flask import Blueprint, jsonify, current_app
+from flask import Blueprint, request, jsonify, current_app
+import logging
 import requests, json
 
 
 bp = Blueprint('southwest', __name__, url_prefix='/')
 
-conf_number = ""
-first_name = ""
-last_name = ""
+conf_number = "W36784"
+first_name = "Jane"
+last_name = "Doe"
 
 @bp.post("/checkin")
+def get_data():
+    if request.is_json:
+        data = request.get_json()
+        # conf_number = request.args.get
+        conf_number = data['conf_number']
+        print(conf_number)
+        return data
```

Comment:
I don't understand why this block isn't working. It complains and gives me this error:

```
TypeError: The view function for 'southwest.get_data' did not return a valid response. The function either returned None or ended without a return statement.
```

Reply:
Added a return on line 20 at the same level as the `if` statement and that took care of the problem.